### PR TITLE
Modify commands list

### PR DESCRIPTION
**Pull request by xiangpengzhao**
This PR:
* Reorder the commands list to make it more clear (I hope so)
* ~~Add shrug command in commands list (I like this command)~~ don't document `¯\_(ツ)_/¯`, keep it as an easter egg. @cblecker :)
* Modify hold command description

/shrug

### CODE REVIEW

File: commands.md [hunk 1]
```diff
@@ -4,32 +4,33 @@
...
 `/unassign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Unassigns specified people (or yourself if no one is specified). Target must already be assigned.
 `/cc [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Request review from specified people (or yourself if no one is specified). Target must be a kubernetes org member.
 `/uncc [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Dismiss review request for specified people (or yourself if no one is specified). Target must already have had a review requested.
-`/area [label1 label2 ...]` | prow [label](./prow/plugins/label) | anyone | adds an area/<> label(s) if it exists
-`/remove-area [label1 label2 ...]` | prow [label](./prow/plugins/label) | anyone | removes an area/<> label(s) if it exists
+`/close` | prow [close](./prow/plugins/close) | authors and assignees | closes the issue/PR
+`/reopen` | prow [reopen](./prow/plugins/reopen) | authors and assignees | reopens a closed issue/PR
+`/hold` | prow [hold](./prow/plugins/hold) | anyone | adds the `hold` label
```
Comment (kargakis) [inline commands.md]: More precisely, it adds a `do-not-merge/hold` label.

File: commands.md [hunk 2]
```diff
@@ -4,32 +4,33 @@
...
 `/release-note` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note` label
 `/release-note-action-required` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note-action-required` label
 `/release-note-none` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note-none` label
-`@kubernetes/sig-<some-github-team>` | prow [label](./prow/plugins/label) | kubernetes org members | adds the corresponding `sig` label
-`/retest` | prow [trigger](./prow/plugins/trigger) | anyone on trusted PRs | reruns failed tests
-`/test all`<br>`/test <some-test-name>` | prow [trigger](./prow/plugins/trigger) | anyone on trusted PRs | runs tests defined in [config.yaml](./prow/config.yaml)
-`/ok-to-test` | prow [trigger](./prow/plugins/trigger) | kubernetes org members | allows the PR author to `/test all`
-`/joke` | prow [yuks](./prow/plugins/yuks) | anyone | tells a bad joke, sometimes
+`/shrug` | prow [shrug](./prow/plugins/shrug) | anyone | adds the `shrug ¯\_(ツ)_/¯` label whenever you want
```
Comment (kargakis) [inline commands.md]: s/shrug ¯\_(ツ)_/¯/ ¯\_(ツ)_/¯/

**Comment by xiangpengzhao** [comment]
@kargakis inline comments addressed. PTAL. Thanks!

File: commands.md [hunk 3]
```diff
@@ -4,32 +4,33 @@
...
 `/release-note` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note` label
 `/release-note-action-required` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note-action-required` label
 `/release-note-none` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note-none` label
-`@kubernetes/sig-<some-github-team>` | prow [label](./prow/plugins/label) | kubernetes org members | adds the corresponding `sig` label
-`/retest` | prow [trigger](./prow/plugins/trigger) | anyone on trusted PRs | reruns failed tests
-`/test all`<br>`/test <some-test-name>` | prow [trigger](./prow/plugins/trigger) | anyone on trusted PRs | runs tests defined in [config.yaml](./prow/config.yaml)
-`/ok-to-test` | prow [trigger](./prow/plugins/trigger) | kubernetes org members | allows the PR author to `/test all`
-`/joke` | prow [yuks](./prow/plugins/yuks) | anyone | tells a bad joke, sometimes
+`/shrug` | prow [shrug](./prow/plugins/shrug) | anyone | adds the `¯\_(ツ)_/¯` label whenever you want
```
Comment (cblecker) [inline commands.md]: shrug is test-infra only.. it's a nice easter egg, but I don't think it should be documented :)

Comment (xiangpengzhao) [reply commands.md]: But I also see it in K/K... https://github.com/kubernetes/kubernetes/pull/52010#issuecomment-327548092

Comment (xiangpengzhao) [reply commands.md]: However, I agree on keeping it as an easter egg :)

**Comment by xiangpengzhao** [comment]
@cblecker PR updated. Didn't document shrug. PTAL. Thanks!

**Comment by BenTheElder** [comment]
I'd love to see #3741 replace this eventually, but for now let's get this in!
Thanks for the cleanup @xiangpengzhao :-)

**Comment by xiangpengzhao** [comment]
@BenTheElder thanks! Agree on generating commands list dynamically.

**Pull request merged**